panel/src/components/Layout/Frame/Frame.vue

```diff
@@ -12,6 +13,7 @@
 	>
 		<slot />
 	</component>
+	<slot v-else />
```

Comment:
This could be controversial.

Reply:
<img width="1271" height="460" alt="Image" src="https://github.com/user-attachments/assets/3c68d06b-c737-41f4-9a6e-0e5b17a1737f" />

Not just controversial, it also breaks stuff.